java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -187,6 +188,10 @@ class ReferenceModeStore private constructor(
     override fun off(callbackToken: Int) {
         containerStore.off(containerCallbackToken)
         callbacks.unregister(callbackToken)
+
+        if (containerStore.closed) {
+            runBlocking { backingStore.clearStoresCache() }
```

Comment:
Unfortunately we can't have `runBlocking` in `arcs.core`, since there is no implementation of it in the common coroutines lib (it's part of the Java-specific one)

Reply:
Oh nooo... is there an alternative way or any suggestions?

Reply:
I saw other runBlocking uses in DirectStore.kt, Arc.kt, etc.

Reply:
nevermind, I changed it to non-suspend call.